scripts/template_matcher.py

```diff
@@ -59,15 +60,63 @@ def _compute_prediction(self, k, img, kp, des):
         kp: keypoints from scene image,   des: descriptors from scene image
         """
 
-        # TODO: find corresponding points in the input image and the templae image
-        #       put keypoints from template image in template_pts
-        #       put corresponding keypoints from input image in img_pts
+        # http://docs.opencv.org/3.0-beta/doc/py_tutorials/py_feature2d/py_matcher/py_matcher.html
+        bf = cv2.BFMatcher()
+        matches = bf.knnMatch(self.descs[k], des, k=2)
 
+        good = []
+        for m, n in matches:
+            if m.distance < 0.75*n.distance:
```

Comment:
Did you end up playing with the good_thresh or ratio_threshold in order to make your method work?  Just curious :)